sos/plugins/ovn_central.py

```diff
@@ -66,13 +85,26 @@ def setup(self):
             'ovn-sbctl get-connection',
         ]
 
+
         schema_dir = '/usr/share/openvswitch'
+        if containerized:
+            cmd = "%s cp %s:/usr/share/openvswitch/ovn*.ovsschema /tmp/" % (
+                self._container_runtime, self._container_name)
```

Comment:
You can't use `/tmp` for this: since the schema file names are predictable this creates a temporary file name race: a malicious user could create files or symlinks here with the names of schema files and use that to mis-direct writes from sos, or to capture incorrect information in the report.

You need to either set up a private subdirectory in `/tmp` (with an unpredictable name and appropriate permissions), or direct the "%s cp" container command to write directly into the sos archive directory.